packages/core/utils/tsconfig.json

```diff
@@ -5,24 +5,24 @@
     "outDir": "./dist",
     "esModuleInterop": true,
     "declaration": true,
+    "noUnusedLocals": true,
     "declarationMap": true,
     "module": "commonjs",
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "sourceMap": true,
     "noImplicitReturns": true,
+    "forceConsistentCasingInFileNames": true,
     "strictNullChecks": true,
     "strictFunctionTypes": true,
     "noImplicitThis": true,
     "allowJs": true,
-    "skipLibCheck": true,
-    "downlevelIteration": true
```

Comment:
Removed `downlevelIteration` since we do not want to compile code to oldest versions of JavaScript engines. 

The `tsconfig.json` file will output code in `ES2021` format, which are supported Node 20. Infact, Node 20 supports many more latest features of ES.

https://node.green/